conda_smithy/templates/circle.yml.tmpl

```diff
@@ -17,8 +19,16 @@ dependencies:
   override:
     - docker pull condaforge/linux-anvil
 
+{% endif -%}
+{% endblock -%}
 test:
   override:
+{%- block env_test %}
+{%- if matrix %}
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
-{# Keep the terminal newline. #}
+{% else %}
+    # In case the skip fails for some reason, do a fast finish.
```

Comment:
What does it mean that it is "skipped"? The automatic rerender should disable the circleci build?

The final file basically only has this comment, so the user doesn't know what this is referencing :-/